### ISSUE TITLE
Request for Full Source Code

### ISSUE THREAD

**Issue by YuukkiFu** [junzastar/DFTr_Voting#4]
Could you please make the training code publicly available？I am very interested in your paper and would like to conduct an in-depth study! I tried to send an email to you for the code, but it seems you have refused to receive it .

**Comment by junzastar** [comment]
Hi, thanks for your interest in our work. You can contact me by my new email address [zachary-jun.zhou@connect.polyu.hk].
I have uploaded some of the necessary code for your reference.

NOTE: Please carefully follow [FFB6D](https://github.com/ethnhe/FFB6D) to prepare your environment, and then you can train or test by running the 'sh' file.

If you have any questions about these codes, please feel free to contact me. Thanks.

**Comment by YuukkiFu** [comment]
I have another question. I would like to ask if the training code for the LM and YCB datasets is the same as MP6D? Or could you please make the training and testing code for these two datasets public?

**Comment by YuukkiFu** [comment]
Thanks!!!  And I have another question. I would like to ask if the training code for the LM and YCB datasets is the same as MP6D? Or could you please make the training and testing code for these two datasets public?

**Comment by junzastar** [comment]
> Thanks!!! And I have another question. I would like to ask if the training code for the LM and YCB datasets is the same as MP6D? Or could you please make the training and testing code for these two datasets public?

Yes, you can use the same code for training/testing, just change the corresponding dataset name.

**Comment by YuukkiFu** [comment]
Thank you for your quick reply! I will give it a try.

**Comment by ZJU-PLP** [comment]
> Thank you for your quick reply! I will give it a try.

@YuukkiFu @junzastar  Hi, I want to know if you have already run this code completely. If so, could you please provide the full evaluation code? I found that parts of the evaluation code are missing—for example, the function “[linear_decode_center_point.](https://github.com/junzastar/DFTr_Voting/blob/main/models/utils_my/pvn3d_eval_utils_kpls_v1.py#L126-L127)” and the files [points_full_uniform_8192.xyz](https://github.com/junzastar/DFTr_Voting/blob/main/models/utils_my/basic_utils.py#L552).
